client/dataEntry.js

```diff
@@ -406,25 +397,17 @@ Template.roundDataEntry.events({
     // We save when the user leaves the currently focused jChester.
     let $jChester = $(e.currentTarget);
 
-    let focusNewJChester = function() {
-      Tracker.afterFlush(function() {
-        let $jChesterNew = $(document.activeElement).closest('.jChester');
-        if($jChesterNew[0] !== $jChester[0]) {
-          $jChesterNew.focus();
-          let $warningIconNew = $jChesterNew.parents("tr").find('i[data-toggle="popover"]');
-          $('[data-toggle="popover"]').not($warningIconNew).popover('hide');
-        }
-      });
-    };
+    Tracker.afterFlush(function() {
+      let $jChesterNew = $(document.activeElement).closest('.jChester');
+      if($jChesterNew[0] !== $jChester[0]) {
+        $jChesterNew.focus();
+        let $warningIconNew = $jChesterNew.parents("tr").find('i[data-toggle="popover"]');
+        $('[data-toggle="popover"]').not($warningIconNew).popover('hide');
```

Comment:
So it looks like this code is going through and hiding the warning popover for all warnings except for the newly focused `jChester`'s warning popover. I'm not sure how much sense this makes until we fix `updateWarningContents` to refer to the data in our `jChester`s rather than the contents of the database.

Also, this should get removed from the `Tracker.afterFlush`, as we're no longer writing to the database, we no longer need to wait for our template to rerender.